yarl/_url.py

```diff
@@ -1034,7 +1034,7 @@ def _parse_host(
         return False, lower_host, is_ascii, "", "", ""
 
     @classmethod
-    def _encode_host(cls, host: str, validate_host: bool = True) -> str:
+    def _encode_host(cls, host: str, validate_host: bool) -> str:
```

Comment:
I was trying to figure out how to move `validate_host` out of here but its tied to `isascii` so its hard to untangle